### DISCUSSION TITLE
Starting web page problem

### DISCUSSION THREAD

**Discussion by amolea** [magicbug/Cloudlog#1036] [General]
Hello,
After installation the program works but the web design is not present. All is like a txt file, no graphics, nothing, just writing with hyperlinks and from time to time some spaces where you can add data. Also when I want to add a new QSO page requests to choose DXCC data from list but list is empty so is not working. Importing ADIF is just fine.
Do you have any clue why it's like this?

![image](https://user-images.githubusercontent.com/52033581/118864475-513bf100-b8e0-11eb-9cb1-2ec0084e9eab.png)

**Comment by GU6EFB-1** [comment]
I have had similar issues 
I had the wrong PHP version must be 7 or above but not 8
I also had wrong permissions on folders and files I set all folder to 771 and all files to 664 and it works fine for me.
Hope this helps but if not I am sure Peter will help you

73 Keith GU6EFB

**Comment by amolea** [comment]
I'm using php ver. 7.4.3 and nginx ver. 1.18.0.
Server is Ubuntu 20.04.2 LTS

BR,
Alex / ON2GXL

**Comment by amolea** [comment]
Hello,
Problem was solved. To solve it I reinstalled all server (I'm using a VM) and I choose apache2. After I configured the apache all worked as expected. Now it seems I have a database error. Below one of the many errors:
A Database Error Occurred
Error Number: 1366

Incorrect integer value: '' for column `cloudlog`.`station_profile`.`station_dxcc` at row 1

INSERT INTO `station_profile` (`station_profile_name`, `station_gridsquare`, `station_city`, `station_iota`, `station_sota`, `station_sig`, `station_sig_info`, `station_callsign`, `station_dxcc`, `station_country`, `station_cnty`, `station_cq`, `station_itu`, `state`, `eqslqthnickname`, `qrzapikey`, `qrzrealtime`) VALUES ('HOME BX', 'JO20FU', 'Evere', '', '', '', '', 'ON2GXL', '', '', '', '17', '24', '', '', '', '0')

Filename: models/Stations.php

Line Number: 71

**Reply by amolea** [reply]
another error, it's not working if I want to add a new QSO. There is no error, just Locator box is squared in red.

**Reply by AndreasK79** [reply]
Did you run admin and update country files?

**Reply by gvjensen** [reply]
When I try and redirect to my https configured server, I get what amolea showed in the first example (even if I redirect to http://192.160.0.100/Cloudlog).

I cannot redirect to http**s**://localhost or http**s**://FQDN/Cloudlog.... 

Ideas?
